committee_admissions/admissions/models.py

```diff
@@ -11,18 +12,45 @@ class Meta:
         proxy = True
 
     @property
-    def is_board_member(self):
-        return bool(self.is_superuser or self.leader_of_committee)
+    def is_privileged(self):
+        """
+        Return true if the user has admission privileges
+        """
+        return bool(self.is_superuser or self.admission_privileges)
+
+    @property
+    def admission_privileges(self):
+        """
+        Return true if the user has the role of LEADER or RECRUTING
+        """
+        correct_role = (
+            Membership.objects.filter(user=self)
+            .filter(Q(role=constants.LEADER) | Q(role=constants.RECRUITING))
+            .first()
+        )
+        if not correct_role:
```

Comment:
Make this more explit. For example a `.exists()` instead of `.first()` and drop the if.